src/index.ts

```diff
@@ -106,7 +106,7 @@ async function main(
   }
 
   let projectName = ''
-  if (target === '.') {
+  if (['.', './'].includes(target)) {
```

Comment:
On Windows, we can also have a value like `.\`, so can you include this too? and also we can change this to a regex check - `const IS_CURRENT_DIR_REGEX = /^(\.\/|\.\\|\.)$/;`

Reply:
Thank you for your review.
I've changed it to provided regex and tested on mac and windows - seems working as exptected:
![image](https://github.com/user-attachments/assets/beeb2939-deb3-49f0-b1cb-aecfb7710950)